### PR TITLE
Add images for Longhorn v1.1.1

### PR DESCRIPTION
**Pull request by cclhsu**
Add images for Longhorn v1.1.1

Signed-off-by: Clark Hsu <clark.hsu@suse.com>

#### Pull Request Checklist ####

- [x] Change does not remove any existing Images or Tags in index-list.txt
- [x] Change does not remove / overwrite exiting Images or Tags in Rancher DockerHub
- [x] New entries are in format `SOURCE DESTINATION TAG`
- [x] New entries are added to the correct section of the list (sorted lexographically)
- [x] New entries have a repo created in Rancher Dockerhub (where the image will be mirrored to)
- [ ] Changes to scripting or CI config have been tested to the best of your ability

### CODE REVIEW

**Review by jenting — APPROVED** [review]
LGTM cc @shuo-wu

**Review by shuo-wu — APPROVED** [review]
LGTM

File: images-list [hunk 1]
```diff
@@ -169,20 +169,30 @@ longhornio/csi-resizer rancher/longhornio-csi-resizer v0.5.1-lh1
 longhornio/csi-resizer rancher/mirrored-longhornio-csi-resizer v0.5.1-lh1
 longhornio/csi-snapshotter rancher/longhornio-csi-snapshotter v2.1.1-lh1
 longhornio/csi-snapshotter rancher/mirrored-longhornio-csi-snapshotter v2.1.1-lh1
+longhornio/backing-image-manager rancher/longhornio-backing-image-manager v1_20210422
```
[the remark this answers is not in the document]
Comment (yasker) [reply images-list:172]: I am a bit confused. In the Longhorn doc, we also have [an air-gap setup guide without Rancher](https://longhorn.io/docs/1.1.1/advanced-resources/deploy/airgap). So I guess that would be fine? 

What did you mean by `the other chart`? Is it the original chart? In that case, if the user installs Longhorn without Rancher, they won't have the images for air gap anyway without downloading the Longhorn images first. So it seems doesn't make a difference if the image is hosted in Rancher Docker Hub or the Longhorn one.

[the remark this answers is not in the document]
Comment (cbron) [reply images-list:172]: I meant the chart in the `master` branch of the rancher/charts repo. Ok if we are fine having rancher airgap only available for rancher's feature charts (and not the original chart in master branch) then its fine as-is.

File: images-list [hunk 2]
```diff
@@ -169,20 +169,30 @@ longhornio/csi-resizer rancher/longhornio-csi-resizer v0.5.1-lh1
...
+longhornio/backing-image-manager rancher/mirrored-longhornio-backing-image-manager v1_20210422
```
[the remark this answers is not in the document]
Comment (cclhsu) [reply images-list:173]: Hi! this image is already setup and available at https://hub.docker.com/r/longhornio/backing-image-manager

[the remark this answers is not in the document]
Comment (cbron) [reply images-list:173]: The original repo exists but not: 
* https://hub.docker.com/r/rancher/mirrored-longhornio-backing-image-manager
* https://hub.docker.com/r/rancher/longhornio-backing-image-manager

**Comment by cbron** [comment]
issue: https://github.com/rancher/rancher/issues/32369

**Review by cbron — APPROVED** [review]

**Pull request merged**